gym/monitoring/monitor.py

```diff
@@ -46,20 +39,8 @@ def capped_cubic_video_schedule(episode_id):
     else:
         return episode_id % 1000 == 0
 
-# Monitors will automatically close themselves when garbage collected
-# (via __del__) or when the program exits (via close_all_monitors's
-# atexit behavior).
-monitors = weakref.WeakValueDictionary()
-def ensure_close_at_exit(monitor):
-    monitors[monitor.monitor_id] = monitor
-
-@atexit.register
-def close_all_monitors():
-    # Explicitly fetch all monitors first so that they can't disappear while
-    # we iterate. cf. http://stackoverflow.com/a/12429620
-    open_monitors = list(monitors.items())
-    for key, monitor in open_monitors:
-        monitor.close()
+def get_running_monitors():
+    return list(monitor_close_registry.close_objects.values())
```

Comment:
I don't love the name `monitor_close_registry` BTW — could we do something like `monitor_closer`?